CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Add hardcoded versioning support, now the exported table
```

Comment:
I don't think we should tell user about implementation ("hardcoding" part). And "exported table" is also confusing, it's the module api.

Reply:
Simplified, also updated PR description